src/components/ApartmentApplicantList/index.js

```diff
@@ -27,66 +32,27 @@ export default class ApplicantList extends Component {
     }
   };
 
-  /**
-   * Creates the Avatar image of the given user
-   *
-   * @param {String} username The username of the desired user image
-   *
-   * @return {String} The profile image of the given user if available
-   */
-  async loadAvatar(username) {
-    // let username = String(profile.AD_Username);
-    try {
-      const { def: defaultImage, pref: preferredImage } = await user.getImage(username);
-      const image = preferredImage || defaultImage;
-      return image;
-    } catch (error) {
-      return null;
-    }
-  }
-
-  renderApplicant(profile) {
-    let avatarImage = this.loadAvatar(profile.AD_Username);
-    return (
-      <ListItem key={profile.AD_Username} className={'applicant-list-item'}>
-        <ListItemAvatar>
-          {avatarImage ? (
-            <Avatar
-              className={`applicant-avatar`}
-              src={`data:image/jpg;base64,${avatarImage}`}
-              sizes="70px"
-            />
-          ) : (
-            <Avatar>
-              <PersonIcon />
-            </Avatar>
-          )}
-        </ListItemAvatar>
-        <ListItemText primary={profile.fullName} secondary={profile.AD_Username} />
-        <ListItemSecondaryAction>
-          {profile.AD_Username === this.props.userProfile.AD_Username ? (
-            <IconButton edge="end" aria-label="remove">
-              <RemoveCircleOutlineIcon color="disabled" />
-            </IconButton>
-          ) : (
-            <IconButton
-              edge="end"
-              aria-label="remove"
-              onClick={this.handleRemove.bind(this, profile)}
-            >
-              <RemoveCircleOutlineIcon color="action" />
-            </IconButton>
-          )}
-        </ListItemSecondaryAction>
-      </ListItem>
-    );
-  }
-
   render() {
     if (this.props.applicants) {
       return (
-        <List className="apartment-applicant-list">
-          {this.props.applicants.map(profile => this.renderApplicant(profile))}
+        <List className="apartment-applicant-list" aria-label="apartment applicants">
+          <ListItem key="applicant-list-header" className="applicant-list-header">
+            <ListItemText primary="Student Applicants" />
+            <GordonPeopleSearch
+              justify="flex-end"
+              disableLink
+              onSearchSubmit={this.handleSelection}
+              Authentication={this.props.Authentication}
+            />
+          </ListItem>
+          {this.props.applicants.map(profile => (
+            <ApplicantListItem
+              key={profile.AD_Username}
+              profile={profile}
+              isPrimaryApplicant={profile === this.props.userProfile}
+              onApplicantRemove={this.handleRemove.bind(this, profile)}
```

Comment:
I have never seen bind be used on anything else than `this`. How does that work?

Reply:
It is used whenever setting a callback on a component to trigger a method of the parent class. If you do not use the `bind` with the callback function, then it will actually call the method every time the component is created/rendered.

Reply:
Okay, so if you did not bind to profile, the application page would re-render every time we removed a profile?

Reply:
No, it would actually to call the `handleRemove` method every time we render the remove button (Which would essentially remove every person as soon as they are added). 

The `bind(this, profile)` part is not binding it to `profile`, it is just binding to `this` and passing `profile` to the `handleRemove` as a parameter.

Reply:
Oh, okay!